matlab/CMakeLists.txt

```diff
@@ -32,8 +110,29 @@ endif()
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake_modules)
 
-# Arrow is Required
-find_package(Arrow REQUIRED)
+# Only build the MATLAB interface C++ tests if MATLAB_BUILD_TESTS=ON.
+if(MATLAB_BUILD_TESTS)
```

Comment:
Could you add `option(MATLAB_BUILD_TESTS ...)` somewhere?
https://cmake.org/cmake/help/latest/command/option.html

Reply:
Good point! Thanks for the suggestion. I'll make this chance.

Reply:
Fixed.